src/components/Navigator.vue

```diff
@@ -195,14 +197,18 @@ export default {
         // get the children
         const { children, ...node } = childrenNodes[index];
         // generate the extra properties
-        const { uid: parentUID = INDEX_ROOT_KEY } = parent || {};
+        const {
+          uid: parentUID,
+          path: parentPath,
+        } = parent;
         // generate a uid to track by
         node.uid = this.hashCode(`${parentUID}+${node.path}_${depth}_${index}`);
         // store the parent uid
         node.parent = parentUID;
         // store the current groupMarker reference
         if (node.type === TopicTypes.groupMarker) {
           node.deprecatedChildrenCount = 0;
+          node.path = `${parentPath}#${anchorize(node.title)}`;
```

Comment:
I think this line could cause potential issues in the situation where DocC is already providing an `anchor` value that we are using for the ID of a given topic group since it's not guaranteed that the value will be the same as what gets generated here.

To avoid any potential broken links, I think we would have to check the `topicSections` page data for `anchor` values to copy here or only generate them using `anchorize` in _both_ places if it isn't already provided there.

Ideally we would also separately update DocC to generate this `path` with the same anchor to avoid the need for manually assigning one here also, although that could be a future improvement.

Reply:
My concerns too, though I dont think docc is emitting IDs for those sections at all. 

https://github.com/apple/swift-docc/blob/61100a442ff1021405e19de8b4b225e576b19cdd/Sources/SwiftDocC/SwiftDocC.docc/Resources/RenderNode.spec.json#L2586-L2614

Reply:
Is this what you are meaning @mportiz08 ? https://github.com/apple/swift-docc-render/pull/456/commits/37ce9f9756b96604d1a7d9f62a83249f015d1ff1

(Test need to be fixed, but just to know your opinion)

Reply:
That is what I was meaning, however I wasn't thinking clearly enough—this will _only_ work for the titles in the navigator that relate to the current page—it won't work for any child pages since we only have the data for the current page to copy the anchors from.

To do this right, I think we would need the navigator index data to include the anchors for all the titles—there isn't a way to do this today and have it work everywhere without that :( sorry

Reply:
Yep, true. I'll revert the latest commit and speak with the backend team to provide us the anchor in the navigator data.

Reply:
Sounds good. To be clear, I don't think any of this will work as it should without that data though. Technically we could change this to assume the navigator will always be vending this data in anticipation of that happening in the future, although I would be a little wary of merging this if that won't be happening for some time. What do you think?

Reply:
I think we should wait for the data to come from the backend before merging this